### ISSUE TITLE
subdir called on a Dist::Zilla::Path object; this will cease to work in Dist::Zilla v7

### ISSUE THREAD

**Issue by soleo** [duckduckgo/p5-dzp-automodulesharedirs#2]
Using `dzil listdeps` in zeroclickinfo-goodies or zeroclickinfo-spices would trigger the following warnings 

```
->subdir called on a Dist::Zilla::Path object; this will cease to work in Dist::Zilla v7; downstream code should be updated to use Path::Tiny API, not Path::Class at /usr/local/perlbrew/perls/5.16/lib/site_perl/5.16.3/Dist/Zilla/Plugin/AutoModuleShareDirs.pm line 35.
->subdir called on a Dist::Zilla::Path object; this will cease to work in Dist::Zilla v7; downstream code should be updated to use Path::Tiny API, not Path::Class at /usr/local/perlbrew/perls/5.16/lib/site_perl/5.16.3/Dist/Zilla/Plugin/AutoModuleShareDirs.pm line 57.
```

Ref https://github.com/DarwinAwardWinner/Dist-Zilla-Plugin-CopyFilesFromBuild/issues/3

**Comment by moollaza** [comment]
@soleo would you be interested in fixing this?

We need to switch to `Path::Tiny` from `Path::Class` in our `Dist::Zilla::Plugin` modules

**Comment by zachthompson** [comment]
@soleo We already have fix here - https://github.com/duckduckgo/p5-dzp-automodulesharedirs/compare/zt/path-tiny - just waiting to update dzil.

**Comment by zachthompson** [comment]
Fixed with release of [v0.002](https://travis-ci.org/duckduckgo/zeroclickinfo-goodies/builds/154534298#L266), e.g. [listdeps](https://travis-ci.org/duckduckgo/zeroclickinfo-goodies/builds/154534298#L272) in recent travis build.
